### DISCUSSION TITLE
Your language needs more exposure!

### DISCUSSION THREAD

**Discussion by ghost** [objeck/objeck-lang#405] [General]
Go to lobste.rs and make an account, then advertise your language. You will get plenty of feedbacks.

Your language is very weird. Everything seems to be there, from the language features to the standard library. At first glance, it seems to be very rock-solid. But in practice, nothing works as expected. This indicated a lack of users and, thus, a lack of testing.

**Comment by objeck** [comment]
Thank you. I am open to suggestions and will check your other feedback in the morning. My approach is somewhat biased (i.e., one-sided); thus, I depend on others to fix my mistakes so the language is more accessible to the broader community.
